src/isar/storage/uploader.py

```diff
@@ -22,21 +70,66 @@ def run(self) -> None:
             inspection: Inspection
             mission_metadata: MissionMetadata
             try:
-                inspection, mission_metadata = self.upload_queue.get(timeout=5)
+                inspection, mission_metadata = self.upload_queue.get(timeout=1)
             except Empty:
+                if self.retry_queue:
```

Comment:
Move this out of the except. I think we want to run  `self._process_retry_queue()` all the time and not only when there is no new inspections. This can potentially lead to a deadlock if we always have a new inspection ready when we check the queue